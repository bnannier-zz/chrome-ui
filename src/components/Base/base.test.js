import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

import { Base, AppBar } from "../";

describe("Base Tests", () => {
  it("Default", function() {
    const wrapper = shallow(
      <Base>
        <AppBar>test</AppBar>
      </Base>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
