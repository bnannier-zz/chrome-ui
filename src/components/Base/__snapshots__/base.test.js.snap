// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base Tests Default 1`] = `
<AppBar
  className=""
>
  test
</AppBar>
`;
